honeybee_vtk/scene.py

```diff
@@ -64,6 +66,15 @@ def assistants(self) -> List[Assistant]:
         """A list of honeybee-vtk assistants working in the scene."""
         return self._assistants
 
+    @property
+    def legend_parameters(self) -> Dict[str, LegendParameter]:
```

Comment:
I wonder if it is more useful to have a `get_legend_param(name)` method to get the legend parameter by name. This can stay but how likely is it that one wants to get all the legend parameters in one go?

Reply:
A scene object hosts legend parameters and following is how a user will apply customization.

```
scene.legend_parameters['Daylight-Factor'].bold = True
scene.legend_parameters['Daylight-Factor'].size = 15
```
This is the reason why the property is spitting out a dictionary. In my opinion, this is a simple approach to apply customization to a legend parameter object at scene level.

Reply:
Not a big fan of this design really. I like the option to get access to the object once and then do whatever you want to do.

```python
df_legend = scene.legend_parameters('daylight-factor')
df_legend.bold = True
df_legend.size = 15
```

The current design encourages creating the dictionary for legend parameters for changing every property of every legend

Reply:
Complied. I created a method to facilitate customization of legend parameters how you like.

> Not a big fan of this design really

I really appreciate such unambiuous and clear feedback.